mobile_config/src/gateway_info.rs

```diff
@@ -153,11 +165,24 @@ pub(crate) mod db {
 
     pub fn all_info_stream<'a>(
         db: impl PgExecutor<'a> + 'a,
+        device_types: &'a [DeviceType],
     ) -> impl Stream<Item = GatewayInfo> + 'a {
-        sqlx::query_as::<_, GatewayInfo>(GET_METADATA_SQL)
-            .fetch(db)
-            .filter_map(|metadata| async move { metadata.ok() })
-            .boxed()
+        match device_types.is_empty() {
+            true => sqlx::query_as::<_, GatewayInfo>(GET_METADATA_SQL)
+                .fetch(db)
+                .filter_map(|metadata| async move { metadata.ok() })
+                .boxed(),
+            false => sqlx::query_as::<_, GatewayInfo>(&DEVICE_TYPES_METADATA_SQL)
+                .bind(
+                    device_types
+                        .iter()
+                        .map(|v| format!("\"{}\"", v))
```

Comment:
I assume the escaping here is required ?  A note to explain why would be useful for future readers, especially given the table schema is managed outside of the oracles

Reply:
The device_types field has a jsonb type but is being used as a string, which forces us adding of the quotes.
![image](https://github.com/user-attachments/assets/668baca6-914b-4510-a718-df01034a4dad)
![image](https://github.com/user-attachments/assets/cec85899-d9f9-4287-bbaf-283df0cadacf)


Column in "meta" database: 
![image](https://github.com/user-attachments/assets/c729f4f0-ae8e-46f0-981c-c90014b48c11)

(Added comment to code)

Reply:
not sure it is a better idea but you can query the field normally like this
`WHERE mhi.device_type #>> '{}' = 'wifiDataOnly'`